public/app/plugins/panel/heatmap-new/utils.ts

```diff
@@ -304,11 +313,10 @@ export function prepConfig(opts: PrepConfigOpts) {
                 // how to expand scale range if inferred non-regular or log buckets?
               }
             }
-            return [dataMin, dataMax];
+            return [yAxisConfig.min ?? dataMin, yAxisConfig.max ?? dataMax];
```

Comment:
does this mean that min can not be zero?

Reply:
hmm, we should guard against <= 0 in the log case

Reply:
but i think your comment was unrelated, maybe you're confusing `||`? :thinking: